controllers/front/OrderReturnController.php

```diff
@@ -34,37 +34,33 @@ class OrderReturnControllerCore extends FrontController
     public $ssl = true;
 
     /**
-     * Initialize order return controller
+     * Initialize order return controller.
+     *
      * @see FrontController::init()
      */
     public function init()
     {
         parent::init();
 
-        $id_order_return = (int)Tools::getValue('id_order_return');
+        $id_order_return = (int) Tools::getValue('id_order_return');
 
         if (!isset($id_order_return) || !Validate::isUnsignedId($id_order_return)) {
             $this->redirect_after = '404';
             $this->redirect();
         } else {
-            $order_return = new OrderReturn((int)$id_order_return);
+            $order_return = new OrderReturn((int) $id_order_return);
             if (Validate::isLoadedObject($order_return) && $order_return->id_customer == $this->context->cookie->id_customer) {
-                $order = new Order((int)($order_return->id_order));
+                $order = new Order((int) ($order_return->id_order));
                 if (Validate::isLoadedObject($order)) {
-                    $state = new OrderReturnState((int)$order_return->state);
-
                     if ($order_return->state == 1) {
                         $this->warning[] = $this->trans('You must wait for confirmation before returning any merchandise.', array(), 'Shop.Notifications.Warning');
                     }
 
                     $this->context->smarty->assign(array(
-                        'orderRet' => $this->getTemplateVarOrderReturn($order_return),
-                        'order' => $order,
-                        'state_name' => $state->name[(int)$this->context->language->id],
-                        'return_allowed' => false,
-                        'products' => $this->getTemplateVarProducts((int)$order_return->id, $order),
-                        'returnedCustomizations' => OrderReturn::getReturnedCustomizedProducts((int)$order_return->id_order),
-                        'customizedDatas' => Product::getAllCustomizedDatas((int)$order->id_cart)
+                        'return' => $this->getTemplateVarOrderReturn($order_return),
+                        'products' => $this->getTemplateVarProducts((int) $order_return->id, $order),
+                        'returnedCustomizations' => OrderReturn::getReturnedCustomizedProducts((int) $order_return->id_order),
+                        'customizedData' => Product::getAllCustomizedDatas((int) $order->id_cart),
```

Comment:
You modified the name of this variable but I do not see where the templates have been updated.